src/h.js

```diff
@@ -1,22 +1,29 @@
-export function h(tag, data) {
-  var node
-  var stack = []
-  var children = []
+function addChild(children, val) {
+  if (val == null || typeof val === 'boolean') return
 
-  for (var i = arguments.length; i-- > 2; ) {
-    stack[stack.length] = arguments[i]
+  if (Array.isArray(val)) {
+    for (var j = 0; j < val.length; j++) {
```

Comment:
@lukeed Oh I'm not aware of this. Do you have a source that highlights this v8 issue? Because in my tests I really don't see any performance impact of not caching it.